src/pipecat/services/rime.py

```diff
@@ -0,0 +1,101 @@
+from typing import AsyncGenerator, Optional
+
+import aiohttp
+from loguru import logger
+from pydantic import BaseModel
+
+from pipecat.frames.frames import (
+    ErrorFrame,
+    Frame,
+    TTSAudioRawFrame,
+    TTSStartedFrame,
+    TTSStoppedFrame,
+)
+from pipecat.services.ai_services import TTSService
+
+
+class RimeTTSService(TTSService):
```

Comment:
We should probably call it `RimeHttpTTSService` so we don't have to rename if they ever support websockets.

Reply:
Good idea. I got a message from the Rime founder today and they're working on a websocket service. I'll update the name since the websocket service will be coming in the future.